README.md

```diff
@@ -6,7 +6,7 @@ Replace WordPress default search with Google Custom Search results.
 
 **Tags:** google, search, cse, custom search engine, programmable search, programmable search engine, google cse, google custom search engine, google programmable search, google programmable search engine, google search
 
-**Requires at least:** 4.9
```

Comment:
Version: 4.9 was added by @\kiranpotphode but I have tested the plugin on 4.8 it is working properly.  requires at least 4.9 is only mentioned in README.md. in readme.txt it is 4.8